datastoretest/mongo.go

```diff
@@ -77,6 +82,14 @@ func (db *DB) Clean() {
 	}
 }
 
+// Drop removes the entire database including all of its collections.
+func (db *DB) Drop() {
```

Comment:
Drop could be performed in Clean directly as I don't see how "Drop" will be distinguished from "Clean" in tests. 

db := NewDatabaseWithHost()
defer db.Clean();